bin/compile.sh

```diff
@@ -30,11 +30,11 @@ fi
 : "${GREP:=grep}"     # Need to update on SunOS
 : "${AWK:=awk}"       # Need to update on SunOS
 
-# At compile time, `_sun_tools_init` is not yet defined.
-if [[ -f "lib/sun_tools.sh" ]]; then
-  # shellcheck disable=1091
-  source "lib/sun_tools.sh" :
-  _sun_tools_init || true
+if [ "$(uname)" = "SunOS" ]; then
+  # ggrep (GNU-grep) is required in SunOS to build and test because the default
+  # implementation of grep in SunOS doesn't support the options `-E` and `-e` 
```

Comment:
I've had a minor fix to address this problem: https://github.com/icy/pacapt/pull/195 and I think you may not need this patch.

Reply:
but I think it's better to include that in your PR too. As I just don't have any sunOS machine here for test :dagger: Can you please do that? Many thanks